src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -417,6 +417,7 @@ def load_arguments(self, _):
         c.argument('ephemeral_os_disk_placement', arg_type=ephemeral_placement_type,
                    help='Only applicable when used with `--size`. Allows you to choose the Ephemeral OS disk provisioning location.')
         c.argument('enable_hibernation', arg_type=get_three_state_flag(), min_api='2021-03-01', help='The flag that enable or disable hibernation capability on the VM.')
+        c.argument('security_type', arg_type=get_enum_type(["TrustedLaunch"], default=None), min_api='2022-11-01', help='Specify the security type of the virtual machine.')
```

Comment:
@AjKundnani Do we currently just need to make the `--security-type` parameter only support `TrustedLaunch`? Or we need to support all security type as `az vm create`?

Reply:
@zhoxing-ms - Currently `TrustedLaunch` only because upgrade of existing VMs to ConfidentialVM is not supported.

Reply:
Got it, thanks~